packages/asset-swapper/src/network/samplers/curve.ts

```diff
@@ -0,0 +1,709 @@
+import { ChainId } from '@0x/contract-addresses';
```

Comment:
Curve is still heinous but at least it's only heinous to other curves now. We can refactor all these configs out into separate files under a curve subdir.